src/components/Modal/ContentModerationModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+
+import Paragraph from "decentraland-gatsby/dist/components/Text/Paragraph"
+import Title from "decentraland-gatsby/dist/components/Text/Title"
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+import { SceneContentRating } from "decentraland-gatsby/dist/utils/api/Catalyst.types"
+import TokenList from "decentraland-gatsby/dist/utils/dom/TokenList"
+import { Button } from "decentraland-ui/dist/components/Button/Button"
+import { Modal } from "decentraland-ui/dist/components/Modal/Modal"
+
+import { PlaceAttributes } from "../../entities/Place/types"
+import RateFlagAdult from "../../images/rating-flag-adult.svg"
+import RateFlagRestricted from "../../images/rating-flag-restricted.svg"
+import RateFlagTeen from "../../images/rating-flag-teen.svg"
+
+import "./index.css"
+
+export type ContentModerationModalProps = {
+  onClickOpen: (e: React.MouseEvent<HTMLElement>, action: boolean) => void
```

Comment:
If the component is a `Modal`, it would be better to keep the props of the base `Modal` component and use `onOpen` and `onClose` instead of `onClickOpen` + a `boolean`